runtime/moonbase/tests/xcm_mock/parachain.rs

```diff
@@ -680,6 +680,54 @@ impl xcm_transactor::Config for Runtime {
 	type AssetTransactor = AssetTransactors;
 }
 
+parameter_types! {
+	pub const MinimumPeriod: u64 = 1000;
+}
+impl pallet_timestamp::Config for Runtime {
+	type Moment = u64;
+	type OnTimestampSet = ();
+	type MinimumPeriod = MinimumPeriod;
+	type WeightInfo = ();
+}
+
+pub struct FixedGasPrice;
```

Comment:
Is this needed? I see in Frontier that feeCalculator is implemented for (), so if the price is not used anywhere we can remove this struct and use () instead?

Reply:
Totally, let me change it